modules/fat.c

```diff
@@ -0,0 +1,131 @@
+/* vim: tabstop=4 shiftwidth=4 noexpandtab
+ * This file is part of ToaruOS and is released under the terms 
+ * of the NCSA / University of Illinois License - see License.md
+ * Copyright (C) 2017 Kevin Lange
```

Comment:
No copyright assignments here, you should replace my name with yours. The only requirement I have is that people keep the same license for things that get upstreamed.